master/getting-started/bare-metal/bare-metal.md

```diff
@@ -39,17 +39,13 @@ Calico supports selector-based policy as normal when running on a gateway or rou
 allowing for rich, dynamic security policy based on the labels attached to your
 workloads.
 
-> **NOTE**
->
-> If you have a host with workloads on it then traffic that is forwarded to
-> workloads bypasses the policy applied to host endpoints. If that weren't the
-> case, the host endpoint policy would need to be very broad to allow all
-> traffic destined for any possible workload.
->
-> Since version 2.1.0, Calico applies host endpoint policy to traffic that is
-> being forwarded between host interfaces.
->
-> ![]({{site.baseurl}}/images/bare-metal-packet-flows.png)
+![]({{site.baseurl}}/images/bare-metal-packet-flows.png)
+
```

Comment:
The diagram here should be part of the Note, I believe, whereas this PR moves it to before the note, so that it looks like part of the main content.  Can it still be part of the note, in the new proposed scheme?  (See https://docs.projectcalico.org/v2.5/getting-started/bare-metal/bare-metal).

Reply:
I don't think it can't be a part of the note, but I can look into this and get back to you.

Reply:
Actually, yes, it fits into the note, so I have put it in there now.